lua/harpoon/ui.lua

```diff
@@ -53,6 +53,11 @@ end
 
 M.toggle_quick_menu = function()
     if win_id ~= nil and vim.api.nvim_win_is_valid(win_id) then
+        local global_config = harpoon.get_global_settings()
+        print(global_config.save_on_menu_quit)
+        if global_config.save_on_toggle == true then
+            vim.cmd(":lua require('harpoon.ui').on_menu_save()")
```

Comment:
I believe you can just call `save_changes()` here without the `vim.cmd()`.

Reply:
that indeed seems to work, not sure why I did it my way in the first place.
Something else, is what I do with writing the global config and then reading the correct property in the if statement the 'correct' way?

Reply:
hah!  

You can just rewrite this as 
```
if global_config.save_on_toggle then
    require('harpoon.ui').on_menu_save()
end
```

No need to check for truth when its true :)
Also no need to call lua _from lua_ *think about it*

Reply:
I already rewrote line 59 to just `save_changes` as Asbjorn suggested. Think it should be all good now. yes?